pymeasure/instruments/instrument.py

```diff
@@ -275,19 +275,21 @@ def binary_values(self, command, header_bytes=0, dtype=np.float32):
         return self.adapter.binary_values(command, header_bytes, dtype)
 
     @staticmethod
-    def control(get_command,  # noqa: C901 accept that this is a complex method
-                set_command,
-                docs,
-                validator=lambda v, vs: v,
-                values=(),
-                map_values=False,
-                get_process=lambda v: v,
-                set_process=lambda v: v,
-                command_process=lambda c: c,
-                check_set_errors=False,
-                check_get_errors=False,
-                dynamic=False,
-                **kwargs):
+    def control(  # noqa: C901 accept that this is a complex method
+        get_command,
+            set_command,
+            docs,
+            validator=lambda v, vs: v,
+            values=(),
+            map_values=False,
+            get_process=lambda v: v,
+            set_process=lambda v: v,
+            command_process=lambda c: c,
+            check_set_errors=False,
+            check_get_errors=False,
+            dynamic=False,
+            **kwargs
+    ):
```

Comment:
This indentation looks weird -- all args should be at the same level. What about this?
```suggestion
    def control(  # noqa: C901 accept that this is a complex method
        get_command,
        set_command,
        docs,
        validator=lambda v, vs: v,
        values=(),
        map_values=False,
        get_process=lambda v: v,
        set_process=lambda v: v,
        command_process=lambda c: c,
        check_set_errors=False,
        check_get_errors=False,
        dynamic=False,
        **kwargs
    ):
```